src/components/victory-scatter/victory-scatter.jsx

```diff
@@ -253,7 +272,37 @@ export default class VictoryScatter extends React.Component {
     return Helpers.evaluateStyle(baseDataStyle, data);
   }
 
-  renderPoint(data, index, calculatedProps) {
+  static getKeyedData(data) {
+    return data.reduce((keyedData, datum, idx) => {
```

Comment:
Yes, that's listed as the "Pull getNodeTransitions out of VictoryScatter..." TODO item above.  Would you like that part of this PR or separate?